mkstage4.sh

```diff
@@ -10,10 +10,20 @@ fi
 #set flag variables to null
 EXCLUDE_BOOT=0
 EXCLUDE_CONNMAN=0
+EXCLUDE_LOST=1
 QUIET=0
+USAGE="usage:\n\
+  `basename $0` [-q -c -b] [-s || -t <target-mountpoint>] <archive-filename> [custom-tar-options]\n\
+  -q: activates quiet mode (no confirmation).\n\
+  -c: excludes connman network lists.\n\
+  -b: excludes boot directory.\n\
+  -l: includes lost+found directory.\n\
```

Comment:
Why do we need long options? we can have `-l` mean exclude the directory, and if the user does not want to exclude it, he just leaves the option out, and we remove `/lost+found` from the default `EXCLUDES`.